xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/user/impl/xwiki/MyFormAuthenticator.java

```diff
@@ -185,6 +188,16 @@ public boolean processLogin(SecurityRequestWrapper request, HttpServletResponse
 
         // process login form submittal
         if ((this.loginSubmitPattern != null) && request.getMatchableURL().endsWith(this.loginSubmitPattern)) {
+
+            // Placed here to not interfere with persistentLoginManager
```

Comment:
I find this comment hard to understand (I don't understand it). Could you explain in comment what is the problem?

Reply:
Would indeed be nice to elaborate a bit on what kind of problem you are avoiding by putting it here. Note that since this is the location where we read user/pass parameters, this is the natural location to put that token validation, so I don't think I would have put a comment explaining why it's here in the first place.

Reply:
In fact, working with the old PR, the token verification was placed at the top of the method. Doing it so caused the verification to fail when persistentLoginManager was intended to be used. Since, it's the natural location you may be right and I should remove this comment.